cmake/install/CMakeLists.txt

```diff
@@ -41,58 +6,14 @@ ENDIF()
 IF(LMMS_BUILD_WIN32 OR LMMS_INSTALL_DEPENDENCIES)
 	# Install system libraries
 	include(InstallRequiredSystemLibraries)
+	include(InstallTargetDependencies.cmake)
 
 	# Collect directories to search for DLLs
 	GET_FILENAME_COMPONENT(QTBIN_DIR "${QT_QMAKE_EXECUTABLE}" PATH)
 	set(LIB_DIRS "${QTBIN_DIR}")
 
 	GET_PROPERTY(PLUGINS_BUILT GLOBAL PROPERTY PLUGINS_BUILT)
 
-	foreach(target lmms ${PLUGINS_BUILT})
-		get_target_property(target_libs ${target} LINK_LIBRARIES)
-
-		foreach(lib ${target_libs})
-			if(TARGET ${lib} OR NOT IS_ABSOLUTE ${lib})
-				continue()
-			endif()
-
-			get_filename_component(lib_dir ${lib} PATH)
-			list(APPEND LIB_DIRS ${lib_dir})
-			if(lib MATCHES ".(lib|dll.a)$")
-				if(IS_DIRECTORY ${lib_dir}/../bin)
-					list(APPEND LIB_DIRS ${lib_dir}/../bin)
-				endif()
-				if(IS_DIRECTORY ${lib_dir}/bin)
-					list(APPEND LIB_DIRS ${lib_dir}/bin)
-				endif()
-			endif()
```

Comment:
This trick should be moved to `cmake/install/InstallTargetDependencies.cmake`, after line 32. It was here to support non-standard locations, ex. `libwinpthread-1.dll` in Travis-CI builds.

Reply:
I tried to solve that with the PREFIXES argument. However, I don't know why it doesn't work.

Reply:
Have you tried adding `../bin` to the `SUFFIXES` argument?